pkg/api/v1beta1/types.go

```diff
@@ -1091,3 +1099,22 @@ type ResourceQuotaList struct {
 	// Items is a list of ResourceQuota objects
 	Items []ResourceQuota `json:"items"`
 }
+
+type Secret struct {
+	TypeMeta `json:",inline"`
+
+	Data map[string][]byte `json:"data,omitempty"`
+	Type SecretType        `json:"type,omitempty"`
```

Comment:
Everywhere else in the API, we use "Kind". For example:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L506

Could we please change this to `SecretKind`?